Ocs dwicinas uioverview subbranch (#379)
osisoft/OCS-Docs: Content_Portal/Overview/get-started/introduction-portal-interface.md

---
uid: introPortalInterface
---

# Introduction to the portal interface

This topic provides a brief overview of the elements you interact with on the OCS portal interface.

When you first access the interface, you see several panes providing information about the state of your system's health as well as quick links to popular documentation topics.

![Portal interface](../images/portal-interface.png "OCS portal interface")

## OCS features

The navigation icon at top left opens a menu from which you display different OCS features. You always have access to this menu wherever you are in the OCS portal.

![Menu icon](../images/menu-icon.png "Menu icon")

> Clicking the navigation icon lists the following features:

Reviewer: I would like to see a screen capture of the menu.

Author: Done

> 

Reviewer: I think this list and description of the menu items needs to be visually pulled together. What do you think about either setting this up as a bulleted list or in a table?

Author: That's fine with me. I did it this way because that seems to be the standard for definition lists. But I have no attachment to this approach.

> Data Management

Provides tools for setting up basic capabilities of OCS, including tenants, connections to PI Systems, the Sequential Data Store (SDS), and metadata rules for data streams

> Data Collection

Provides mechanisms to bring data from multiple sources and systems into a namespace within OCS

> Visualization

> Allows you to view data trends and use assets to set up digital twins of real world physical entities

Reviewer: ```suggestion
Allows you to view data trends and use assets to set up digital twins of real-world physical entities
```

Author: Done

> Analytics

Provides tools for shaping and querying large datasets

> Security

Allows you to specify identity providers for authentication and add users, roles, and clients to your tenant

> Developer Tools

Provides code samples, an API console, and an editor for the OSIsoft Message Format (OMF)

> Support

> Offers access to documentation, logs, support links, an OCS blog, and other useful information

Author: Done

## Feature pages

Clicking one of the menu items in the list of OCS features takes you to a page for that feature.

![Feature details](../images/feature-details.png "Feature details")

Most pages consist of two panes:

- The left pane is where you perform the tasks associated with the OCS resource. For example, the left pane is where you add and edit or manage permissions for the OCS resource.
- The right pane has one or more tabs:   
  - The Details tab contains additional information about the selected resource.
  -  The Information tab provides documentation about the current feature and describes how to perform relevant tasks.
  - Some resources have additional panes for other configuration tasks.

## Notifications

At top right, the View Notifications icon lists any OCS-generated notifications.

![Notifications icon](../images/notifications-icon.png "Menu icon")

## User and tenant information

> Also at top right, a drop-down menu gives you access to information specific to your own user and tenant. This menu provides:

Reviewer: ```suggestion
At top right, a drop-down menu gives you access to information about your user profile and tenant. This menu provides:
```

Author: Done

- Tenant information
- Resource usage data
- User profile information and roles assigned to your user

> The menu also lets you enter feedback about OCS.

Reviewer: ```suggestion
The menu also provides a link to page where you can give feedback about OCS.
```
Or something like that

Author: Done
